<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FunctionVariableValueDialog</class>
 <widget class="QDialog" name="FunctionVariableValueDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>变量取值范围与样本</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTableWidget" name="tableVariables">
     <column>
      <property name="text">
       <string>变量</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>类型</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>目标值/约束</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>典型值</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>可行范围</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>SAT 样本（;分隔）</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutButtons">
     <item>
      <widget class="QPushButton" name="btnAddRow">
       <property name="text">
        <string>新增变量</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnRemoveRow">
       <property name="text">
        <string>删除所选</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSyncConstraints">
       <property name="text">
        <string>同步输入约束</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>

